src/test/java/org/opentripplanner/updater/vehicle_parking/VehicleParkingAvailabilityUpdaterTest.java

```diff
@@ -0,0 +1,161 @@
+package org.opentripplanner.updater.vehicle_parking;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.opentripplanner.transit.model._data.TransitModelForTest.id;
+
+import com.google.common.util.concurrent.Futures;
+import java.time.Duration;
+import java.util.List;
+import java.util.concurrent.Future;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.framework.geometry.WgsCoordinate;
+import org.opentripplanner.framework.i18n.I18NString;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.routing.vehicle_parking.VehicleParking;
+import org.opentripplanner.routing.vehicle_parking.VehicleParkingService;
+import org.opentripplanner.routing.vehicle_parking.VehicleParkingSpaces;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+import org.opentripplanner.transit.service.TransitModel;
+import org.opentripplanner.updater.GraphUpdaterManager;
+import org.opentripplanner.updater.GraphWriterRunnable;
+import org.opentripplanner.updater.spi.DataSource;
+import org.opentripplanner.updater.spi.GraphUpdater;
+
+class VehicleParkingAvailabilityUpdaterTest {
+
+  private static final VehicleParkingUpdaterParameters PARAMETERS = new VehicleParkingUpdaterParameters() {
+    @Override
+    public VehicleParkingSourceType sourceType() {
+      return VehicleParkingSourceType.SIRI_FM;
+    }
+
+    @Override
+    public UpdateType updateType() {
+      return UpdateType.AVAILABILITY_ONLY;
+    }
+
+    @Override
+    public Duration frequency() {
+      return Duration.ZERO;
+    }
+
+    @Override
+    public String configRef() {
+      return null;
+    }
+  };
```

Comment:
Would it be possible to use the `newNodeAdapterForTest` to load in the parameters instead of this?

Reply:
Done in https://github.com/opentripplanner/OpenTripPlanner/pull/5979/commits/1837b79b2390d4a44f4c536c1aa19d11a048c5e8